mixer/pkg/cache/ttlCache.go

```diff
@@ -108,18 +109,17 @@ func (c *ttlCache) evictExpired(t time.Time) {
 	n := t.UTC().UnixNano()
 	atomic.StoreInt64(&c.baseTimeNanos, n)
 
-	var count uint64
-
 	c.entries.Range(func(key interface{}, value interface{}) bool {
 		e := value.(*entry)
 		if e.expiration <= n {
 			c.entries.Delete(key)
-			count++
 		}
 		return true
 	})
+}
 
-	atomic.AddUint64(&c.stats.Writes, count)
```

Comment:
Not sure why are we not recording stats for evict methods ?

Reply:
I was having trouble doing this in a meaningful way for all edge cases. So I decided to only track explicit sets/gets.